include/irr/asset/NblAssetCompileConfig.h

```diff
@@ -0,0 +1,204 @@
+// Copyright (C) 2018-2020 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
+
+#ifndef __NBL_COMPILE_CONFIG_H_INCLUDED__
+#define __NBL_COMPILE_CONFIG_H_INCLUDED__
+
+//! Irrlicht SDK Version
+#define NABLA_VERSION_MAJOR 0
+#define NABLA_VERSION_MINOR 3
+#define NABLA_VERSION_REVISION 0
+// This flag will be defined only in SVN, the official release code will have
+// it undefined
+//#define IRRLICHT_VERSION_SVN -alpha
+#define NABLA_SDK_VERSION "0.3.0-beta2"
+
+#define NEW_MESHES
+
+#include <stdio.h> // TODO: Although included elsewhere this is required at least for mingw
+
+//! Passing defines to the compiler which have NO in front of the _NBL definename is an alternative
+//! way which can be used to disable defines (instead of outcommenting them in this header).
+//! So defines can be controlled from Makefiles or Projectfiles which allows building
+//! different library versions without having to change the sources.
+//! Example: NO_NBL_COMPILE_WITH_X11_ would disable X11
```

Comment:
remove this comment, everything should be `settable/unsettable` from the CMake GUI options